tools/clone_fileset/main/main.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"flag"
+	"io"
+	"os"
+
+	"github.com/m3db/m3db/persist/encoding/msgpack"
+	"github.com/m3db/m3db/persist/fs"
+	"github.com/m3db/m3db/ts"
+	xlog "github.com/m3db/m3x/log"
+	xtime "github.com/m3db/m3x/time"
+)
+
+const (
+	defaultBufferSize = 4096
```

Comment:
Let's use 64kb that gives a slight boost in write perf, also its what we use as default settings for M3DB and in production.  That is 65536 here.